scene/3d/area_3d.h

```diff
@@ -163,6 +167,18 @@ class Area3D : public CollisionObject3D {
 	void set_priority(real_t p_priority);
 	real_t get_priority() const;
 
+	void set_wind_force_magnitude(real_t p_wind_force_magnitude);
+	real_t get_wind_force_magnitude() const;
+
+	void set_wind_attenuation_factor(real_t p_wind_attenuation_factor);
+	real_t get_wind_attenuation_factor() const;
+
+	void set_wind_source(Vector3 p_wind_source);
```

Comment:
The argument could be `const Vector3 &p_wind_source` to avoid a copy (but based on my previous suggestion the type will probably change to `const NodePath &p_wind_source_path`)